### DISCUSSION TITLE
Backbone Border Router

### DISCUSSION THREAD

**Discussion by kmfnjtze8888** [openthread/openthread#7671] [Q&A] [answered]
Hello,

I wanted to test the backbone border router feature so I set OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE to 1 on ot-br-posix for my raspberry pi but this seems to be causing an issue with my rcp. Typing in "sudo ot-ctl" says: connect session failed: No such file or directory. Also, the otbr-agent.service keeps resetting.

Would I need to also enable the BBR feature for my rcp for this to work? I am using silicon labs' brd4170a as my rcp, and raspberry pi 3b+ as my border router host.

**Comment by jwhui** [comment, marked as answer]
For Raspberry Pi, the Backbone Router feature should be enabled by default - see [examples/platforms/raspbian/default](https://github.com/openthread/ot-br-posix/blob/6a7a8c62e8df846f460e9ffe0419a7bbca1b5560/examples/platforms/raspbian/default#L35).

The `otbr-agent` process not starting typically indicates that it is not able to connect to the RCP or that the RCP version is incompatible. The OTBR logs should provide more visibility into the particular reason.
